xdsl/dialects/builtin.py

```diff
@@ -1875,6 +1875,26 @@ def tensor_from_list(
         t = TensorType(data_type, shape)
         return DenseIntOrFPElementsAttr.from_list(t, data)
 
+    def get_values(self) -> Sequence[int] | Sequence[float]:
+        """
+        Return all the values of the elements in this DenseIntOrFPElementsAttr
+        """
+        return tuple(el.value.data for el in self._data.data)
+
+    def get_attrs(self) -> Sequence[AnyIntegerAttr] | Sequence[AnyFloatAttr]:
+        """
+        Return all elements of the dense attribute in their relevant
+        attribute representation (IntegerAttr / FloatAttr)
+        """
+        return self._data.data
+
+    def is_scalar(self) -> bool:
```

Comment:
Can we rename this to `is_splat`?
`is_scalar` would mean that there is a single element in the tensor, where here we want to say that all elements are the same.

Reply:
Okay, thanks!